assets/js/vue/FileManager.vue

```diff
@@ -354,6 +354,11 @@ const initDropzone = () => {
     maxFilesize: null,
     clickable: '#upload-file-button',
     timeout: 0,
+    error: function error(file, message) {
```

Comment:
The third parameter is `xhr` you can get the Status from it, that may be better than checking the message.code, because if something else went wrong, you might not get a message, and then BOOM!